eng/pipelines/coreclr/templates/run-performance-job.yml

```diff
@@ -64,12 +64,14 @@ jobs:
     - ${{ if eq(parameters.runtimeType, 'wasm') }}:
       - HelixPreCommandsWasmOnLinux: >-
           sudo apt-get -y remove nodejs &&
-          curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash - &&
+          curl -fsSL https://deb.nodesource.com/setup_18.x | sudo -E bash - &&
```

Comment:
We should consider making a note that this needs to be updated every time we bump our linux version

Reply:
I'll add this in a follow up PR.